README.md

```diff
@@ -1,2 +1,49 @@
+# Introduce junbae
+
+We introduce Junbae for people using the this vscode extenstion.
+We don't know the gender yet, but i think it will be a pretty cute parrot who will be raised by the person making this program.
+The name is taken from Junbae's name.
+
+# Motivation
+
+We wanna create vscode extension program that does not effective for coding but gives you the satisfication of coding while typing your code.
+It is sure you have satisfication for visual effect but you does not get the help typing code.
+We are referrence to vscode's PowerMode extension program which has been unmaintained since 2017.
```

Comment:
It's different character between Power Mode and our mode. cuz Power Mode's concept is `power`. they shake the screen, set off fireworks ... etc. but our case, more relaxable, or calm.

There is one thing the same with us, just typing effect. so, it's not necessary sentence i think.